data/libwacom.stylus

```diff
@@ -1,21 +1,21 @@
 # Some generic fallback styli
 [0xfffff]
 Name=General Pen
+PairedIds=0xffffe;
```

Comment:
my bikeshed here is that we should use a different name. `PairedId` is already used for tablet files and I'd rather not have any accidental overlap here. 

"well, what's a better name then", you ask? true, you got me there... `PairedStylusId` at least to have the clear differentiation.

Reply:
ACK